### PR TITLE
CQ-4356927 UI Tests in WKND project fails occasionally

### PR DESCRIPTION
**Pull request by mateusz-kochanek**
## Description

Fixed UI tests. Added rewrite from default vhost to wknd vhost.

## Related Issue

https://jira.corp.adobe.com/browse/CQ-4356927

## Motivation and Context

Encountering pipeline issue. UI testing on WKND fails.

## How Has This Been Tested?

Successfully run wknd-sandboxes-full-stack pipeline.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: dispatcher/src/conf.d/available_vhosts/wknd.vhost [hunk 1]
```diff
@@ -135,6 +135,10 @@ Include conf.d/variables/custom.vars
 	<IfModule mod_rewrite.c>
 		RewriteEngine	on
 		Include conf.d/rewrites/rewrite.rules
+
```
Comment (bisswanger) [inline dispatcher/src/conf.d/available_vhosts/wknd.vhost:138]: can you maybe revert the changes in this file?

They might be needed in some cases but could cause issues in others.
Also the right place rather would be to include the rewrite rule in the *.rules file

But this is not related to UI testing, so we can leave that part as is

**Review by SachinMali — APPROVED** [review]
